dotCMS/src/main/java/com/dotcms/jobs/business/processor/impl/ImportContentletsProcessor.java

```diff
@@ -325,11 +326,12 @@ public Map<String, Object> getResultMetadata(Job job) {
      * @param preview          Flag indicating whether the operation is a preview or publish
      * @param job              The import job configuration
      * @param fileToImport     The CSV file to be imported
+     * @param fileTotalLines   The total number of lines in the file being processed
      * @param user             The user performing the import
      * @param progressCallback Callback for tracking import progress
      */
     private void handleImport(final boolean preview, final Job job, final File fileToImport,
```

Comment:
more than three parameters smell bad, would consider to group into a bean

Reply:
This is related with how the `ImportUtil` already works and it is outside the scope of the ticket.